tests/upgrades/test_errata.py

```diff
@@ -109,6 +109,17 @@ def _host_status(self, client_container_name=None):
             query={'search': '{0}'.format(client_container_name)})
         return host
 
+    def _errata_count(self, ak):
+        """ fetch the content host details.
+         :param: str ak: The activation key
+        :return: int errata_count : errata count
+        """
+        host = entities.Host().search(query={
+            'search': 'activation_key={0}'.format(ak.name)})[0]
+        applicable_errata_count = host.content_facet_attributes[
+            'errata_counts']['total']
```

Comment:
Just a query, does it fetch the applicable errata only ? or both applicable and installable ?

Reply:
As per offline discussion it is an installable errata and bydefault applicable errata. I will make change here renamed it to installable.